.expeditor/bundle_install.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+set -evx
+
+gem environment
+bundler_version=$(grep bundler omnibus_overrides.rb | cut -d'"' -f2)
+gem install bundler -v $bundler_version --user-install --conservative
+# WITH: ci (for version bumping and changelog creation)
+export BUNDLE_WITHOUT=omnibus_package:test:pry:integration:docgen:maintenance:travis:aix:bsd:linux:mac_os_x:solaris:windows:development
+bundle _${bundler_version}_ install
```

Comment:
just do `bundle install` here -- you've already asserted the bundler version you're going to be running is the correct one, and this seems to get into a fight with the 1.14.x trampolining code.